src/components/customObjects/Song.js

```diff
@@ -0,0 +1,3 @@
+export default class Song {
+    //add neccassary fields from db here
+}
```

Comment:
should custom data objects be in /components? or common or src?

Reply:
actually probably shouldn't be in components maybe in src/CustomClasses folder

Reply:
fixed in next commit